lib/dynamics_crm/xml/criteria.rb

```diff
@@ -2,11 +2,14 @@ module DynamicsCRM
   module XML
 
     class Criteria < Array
+      SUPPORTED_OPERATORS = %w(And Or)
 
       attr_accessor :filter_operator
-      def initialize(tuples=[])
-        super
-        @filter_operator = 'And'
+      def initialize(tuples=[], filter_operator=nil)
```

Comment:
Can we make this a named parameter and should it default to `And`? 
`def initialize(tuples=[], filter_operator: 'And')`